packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md

```diff
@@ -4,15 +4,15 @@ excerpt: Learn how to interact with Okta APIs by using scoped OAuth 2.0 access t
 layout: Guides
 ---
 
-This guide shows you how to interact with Okta APIs by using scoped OAuth 2.0 access tokens.
+This guide explains how to interact with Okta APIs by using scoped OAuth 2.0 access tokens.
 
 ---
 
 **Learning outcomes**
 
 * Define allowed scopes for your app.
 * Get an access token and make an API request.
-* Learn about the available scopes and supported endpoints.
+* Know what are the available scopes and supported endpoints.
```

Comment:
Maybe change the first H2 (below) to "About OAuth 2.0 for Okta APPI Endpoints" to match the others?

Reply:
👍